eng/Build.proj

```diff
@@ -35,7 +35,7 @@
 
     <ItemGroup>
       <BuildProject Include="$(RepoRoot)src/**/*.csproj;$(RepoRoot)src/**/*.vbproj" Condition="'$(SrcProjects)' == 'true'" />
-      <BuildProject Include="$(RepoRoot)tests/**/*.csproj;$(RepoRoot)src/**/*.vbproj" Condition="'$(TestProjects)' == 'true'" />
+      <BuildProject Include="$(RepoRoot)tests/**/*.csproj;$(RepoRoot)tests/**/*.vbproj" Condition="'$(TestProjects)' == 'true'" />
```

Comment:
Ohhh. Was this a bug? It is likely something I cause via copy-paste.

Reply:
Yeah. I noticed that one of our unit test projects (Microsoft.VisualStudio.Editors.UnitTests.vbproj) never produced results with build.cmd, and then I found this. :-)